packages/@okta/vuepress-site/docs/guides/sign-into-web-app/define-callback/go/define-route.md

```diff
@@ -0,0 +1,107 @@
+To set up the callback route, you need to define a handler:
+
+```go
+http.HandleFunc("/authorization-code/callback", AuthCodeCallbackHandler)
+```
+
+Then we need to define how the handler works. During this, we exchange the code for an access token and id token.
```

Comment:
"Then we need to" -> "Then you need to"
In previous step, you've use "you need to" :)